src/components/PathwaysList/PathwayModal.tsx

```diff
@@ -16,18 +16,21 @@ import shortid from 'shortid';
 import { usePathwayContext } from 'components/PathwayProvider';
 import useStyles from './styles';
 import { createNewPathway } from 'utils/builder';
+import { Pathway } from 'pathways-model';
 
-interface NewPathwayModalProps {
+interface PathwayModalProps {
   open: boolean;
   onClose: () => void;
+  editPath?: Pathway;
```

Comment:
Suggest this be renamed to something which includes the entire name "pathway". The identifier "path" doesn't have meaning in the builder but in Camino "path" refers to the actual nodes the patient was on whereas "pathway" is the entire datastructure